src/esp32/tls.rs

```diff
@@ -48,13 +48,13 @@ pub struct Esp32TlsStream {
 }
 
 pub struct Esp32TlsServerConfig {
-    srv_cert: &'static [&'static [u8]],
+    srv_cert: [Vec<u8>; 2],
     srv_key: *const u8,
     srv_key_len: u32,
 }
 
 impl Esp32TlsServerConfig {
-    pub fn new(srv_cert: &'static [&'static [u8]], srv_key: *const u8, srv_key_len: u32) -> Self {
+    pub fn new(srv_cert: [Vec<u8>; 2], srv_key: *const u8, srv_key_len: u32) -> Self {
```

Comment:
Would it be useful to put in a comment explaining what the two items in the cert array are? Maybe it's obvious if you're familiar with this stuff, but at first glance I don't see it (maybe one is the public key and one is the private key?).

Reply:
yeah, I'll add a comment